galasa-ui/src/app/page.tsx

```diff
@@ -23,8 +26,10 @@ export default function HomePage() {
 
   return (
     <div id="content">
+      <PageTile data-testid="page-tile" title={"Home"} />
```

Comment:
`data-testid` can be easily overused and ends up littering the code with code specific to tests, is there another way we can grab an element for testing purposes?